views/auth/signup.handlebars

```diff
@@ -25,29 +25,34 @@
         <div class="form-text">PASSWORD REQUIREMENTS HERE</div>
     </div>
 
+    <div class="mb-3">
+        <label id="password-confirmation-label" for="password-confirmation" class="form-label">Confirm Password</label>
+        <input id="password-confirmation" type="password" name="password-confirmation" class="form-control" />
+    </div>
```

Comment:
Need new issue to track the work for check passwords match

Reply:
I don't think the name can have a hyphen in it since it's being passed in the req.body. I might be wrong

Reply:
https://github.com/dwilliamson5/CS546_WS_final_project/issues/41

Reply:
Gonna finish reviewing your PR next so that hopefully some of the profile stuff can be copied here